docs(fix): updated tutorials MTA-1684 (#400)
scaleway/docs-content: tutorials/cloud-controller-manager/index.mdx

The goal of this step is to create a `Kubernetes` cluster using `kubeadm` on Scaleway Elements [Instances](https://www.scaleway.com/en/virtual-instances/).

> For the purpose on this example, you will need to create three Ubuntu Bionic Instances:

Reviewer: ```suggestion
To follow this example, you need to create three Ubuntu Bionic Instances:
```

3. During the installation the `kubeadm join` command displays. Note it down as it is required for the worker nodes to join the cluster. You can also copy the `kubeconfig` file and save it on your local computer.

> 4. Execute the `kubeadm join` command on each of the worker nodes to join the cluster:

Reviewer: ```suggestion
4. Execute the `kubeadm join` command on worker node to join the cluster:
```

</Message>

> 1. Create a `k8s-scaleway-secret.yml` containing the following information:

Reviewer: ```suggestion
1. Create a `k8s-scaleway-secret.yml` file containing the following information:
```
